src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/AnnotationThreadContent.test.js

```diff
@@ -32,17 +32,26 @@ jest.mock('../useAnnotationThread', () => {
     }));
 });
 
+jest.mock('../../../../common/annotator-context/useAnnotatorEvents', () => {
```

Comment:
Should we declare this mock in `common/annotator-context/__mocks__/useAnnotatorEvents` so it can be reused?

Reply:
it was so easy to mock here. I thing creating common mock would be too much hustle for just 2 places. If that would be used in more places then that would be worth while